metrix/migrations/2019-11-05-003436_create_metrics_table/up.sql

```diff
@@ -0,0 +1,8 @@
+-- Your SQL goes here
+CREATE TABLE metrics (
+    id SERIAL PRIMARY KEY,
+    metric_name VARCHAR(255) NOT NULL,
+    data JSON NOT NULL,
```

Comment:
So we'll need a QL of some kind then?

Reply:
Yea, I was thinking of basically allowing json path sectors. But willing to chat through an alternate solution vision.